index.js

```diff
@@ -40,14 +41,21 @@ function httpCheck(module, url, cb) {
 
 var checks = {
   tcp: function (url, cb) {
-    net.connect({
+    var timeout = setTimeout(function () {
+      socket.destroy()
+      cb(null, false, true)
+    }, DEFAULT_TIMEOUT);
```

Comment:
Any reason not to use `socket.setTimeout(DEFAULT_TIMEOUT, function () {})` here?